setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
could you make this file be called `setup` (no `.sh` suffix) & similar for the other scripts